code/controllers/configuration/entries/game_options.dm

```diff
@@ -453,6 +453,23 @@
 /datum/config_entry/number/max_shuttle_size
 	config_entry_value = 250
 
+/datum/config_entry/flag/save_shuttle_ruins
+	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/string/shuttle_ruin_filepath
+	config_entry_value = "data/customShuttleSaves/"
+	protection = CONFIG_ENTRY_LOCKED
+
+/datum/config_entry/number/shuttle_total_filesize_max
+	config_entry_value = 1000000
```

Comment:
I feel like these should be in KB or MB in the config and be converted, so people don't have to type the number out and end up being off by a zero.